Cargo.toml

```diff
@@ -22,20 +22,20 @@ either = "1.13"
 futures = "0.3"
 hex = "0.4.3"
 isomdl = { git = "https://github.com/spruceid/isomdl", rev = "1f4f762" }
-oid4vci = { git = "https://github.com/spruceid/oid4vci-rs", rev = "020887f" }
+oid4vci = { git = "https://github.com/spruceid/oid4vci-rs", branch = "ssi-0.9" }
```

Comment:
Please use a `rev` still. Otherwise things can get out of sync between peoples machines if new commits are pushed to the `ssi-0.9` branch.